src/app/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+# 
+class Errand(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    id = models.SmallIntegerField(primary_key=True)
+    title = models.CharField(max_length=20)
+    priority = models.SmallIntegerField()
+    # address fields
+    streetaddr = models.CharField(max_length=30)
+    city = models.CharField(max_length=20)
+    state = models.CharField(max_length=2)
+    zip = models.SmallIntegerField() # more for last 4?
+    # time variables
+    start = models.DateTimeField(default=0) # needs to change
+    end = models.DateTimeField(default=0) # needs to change
```

Comment:
I think that we want these fields to be optional since they aren't set by default and I'm not sure 0 will work as the default in a datetime field.